content/docs/Quick starts/quick-start-linux.md

```diff
@@ -40,6 +41,9 @@ must do one of the following:
     the repository:
 
         git clone https://github.com/AcademySoftwareFoundation/OpenCue.git
+    
+3. Make sure you have the ```python-dev package``` installed on your system corresponding to the Python version you
```

Comment:
I don't think we need this added step here -- if we've mentioned it in the "Before you begin" section, that's good enough, it's just redundant here.

Reply:
I have removed the redundant step in the new commit, please verify, thanks.